platform/ios/app/MBXUserLocationAnnotationView.h

```diff
@@ -0,0 +1,5 @@
+#import "MGLUserLocationAnnotationView.h"
```

Comment:
You should be able to just import the Mapbox umbrella header or module. But this PR is missing changes to Mapbox.h and the project file that would expose MGLUserLocationAnnotationView.h publicly.